packages/svelte/messages/compile-errors/style.md

```diff
@@ -22,9 +22,9 @@
 
 > A :global {...} block cannot modify an existing selector
 
-## css_global_block_invalid_placement
+## css_global_block_invalid_modifier_start
 
-> A :global {...} block can only appear at the end of a selector sequence (did you mean to use :global(...) instead?)
+> A :global {...} block at the very beginning cannot be modified by other selectors
```

Comment:
We're going to need to rewrite all the ':global {...} block' messages to account for `:global .whatever`

```suggestion
> A `:global` selector can only be modified if it is a descendant of other selectors
```

Reply:
Either Github played a trick on you or you commented right before I pushed changes in that direction - either way, I adjusted the messages